tauri/src/db/mod.rs

```diff
@@ -45,6 +45,26 @@ impl DB {
         Ok(db)
     }
 
+    pub async fn save_balances(&self, balances: AlchemyResponse, chain_id: u32) -> Result<()> {
+        let mut conn = self.tx().await?;
+
+        for balance in balances.token_balances {
+            if !balance.token_balance.is_zero() {
```

Comment:
I added this check here, but this seems to me a bad idea. I think this check should be made on the front end side, here we should always add to the DB.

Reply:
well, the check here will just be wrong.
e.g.: if you have 10 USDC and then transfer them out, the 2nd transaction won't go through here, and the UI will still how 10 USDC in your balance

I would store everything here. the decision of whether to show tokens with zero balances is a different feature (probably a combination of having an option in the UI, and also some automatic filter to hide the spam coins)